### DISCUSSION TITLE
@prisma/client did not initialize yet. Please run "prisma generate" and try to import it again.

### DISCUSSION THREAD

**Discussion by Viraj3021** [prisma/prisma#23984] [Q&A]
### Bug description

![image](https://github.com/prisma/prisma/assets/85681437/8cd0bb6c-6ecd-425e-abf8-143ff7546ea5)

### Prisma information

```prisma
// Add your schema.prisma
```

```ts
// Add your code using Prisma Client
```

### Environment & setup

- OS: 
- Database: 
- Node.js version: 

### Prisma Version

```

```

**Comment by ludralph** [comment]
Hi @Viraj3021 

Can you please run `npx prisma generate` and try executing the command again to run the script?

**Comment by Viraj3021** [comment]
Hi @ludralph 
Great, it worked! Thank you for your help. The issue has been resolved.
